deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -253,6 +242,7 @@ spec:
           - watch
           - update
           - patch
+          - delete
```

Comment:
This will go away in this patch once https://github.com/infrawatch/service-telemetry-operator/pull/482 lands and is merged here.